css/style.css

```diff
@@ -845,17 +854,11 @@ button:hover {
 
 @media (max-width: 480px) {
 	/* hide board actions on mobile */
-	#board-actions {
+	#board-header-controls {
 		display: none !important;
```

Comment:
@pixelipo Yes, having a menu makes sense, but we should keep dedicated buttons if there is enough space.